api/v1alpha1/oadp_types.go

```diff
@@ -97,6 +97,9 @@ type VeleroConfig struct {
 	// Specify whether CSI snapshot data should be moved to backup storage by default
 	// +optional
 	SnapshotMoveData *bool `json:"snapshotMoveData,omitempty"`
+	// Disable informer cache for Get calls on restore. With this enabled, it will speed up restore in cases where there are backup resources which already exist in the cluster, but for very large clusters this will increase velero memory usage. Default is false.
```

Comment:
flag name is disable but explanation says if enabled. would change "enabled" to "true" or "false" (if description in velero also says disable and enable, then better to let equal for consistency)

Reply:
yeah grabbed the description from upstream velero: https://github.com/vmware-tanzu/velero/blob/1034d6aee0b32b6ec0dbe4203902cd1d465ec950/pkg/cmd/server/server.go#L239